packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -179,4 +183,18 @@ flutter:
     });
   });
 
+  test('Failed directory delete shows message', () async {
```

Comment:
Why not use testUsingContext or the testbed?

Reply:
I'm experimenting a bit with reducing our usage of zone values for testing